manipulate_wheels.py

```diff
@@ -179,27 +180,45 @@ def main():
                             numpy_req_found = True
                             if args.verbose:
                                 print('Found numpy dependency.')
-                            curr_version_req = curr_req.replace("numpy","").replace("(","").replace(")","").strip()
+                            curr_version_req = curr_req.split(';')[0].replace("numpy","").replace("(","").replace(")","").strip()
                             req_tokens = ["numpy",""]
+                            if len(curr_req.split(';')) == 2:
```

Comment:
This code is a bit hard to read, and you split curr_req 3 times.
`if ';' in curr_req` ? Or
```
curr_reqs = curr_req.split(';')
curr_version_req = curr_reqs[0].replace("numpy","").replace("(","").replace(")","").strip()
req_tokens = ["numpy",""]
if len(curr_reqs) == 2:
    req_tokens.append(';' + curr_reqs[1])
```

Reply:
should be better now